src/test/java/seedu/letsgethired/logic/parser/NoteCommandParserTest.java

```diff
@@ -22,15 +23,16 @@ public class NoteCommandParserTest {
     public void parse_indexSpecified_success() {
         // have note
         Index targetIndex = INDEX_FIRST_APPLICATION;
-        String userInput = targetIndex.getOneBased() + " " + PREFIX_NOTE + VALID_NOTE_JANE_STREET;
-        NoteCommand expectedCommand = new NoteCommand(INDEX_FIRST_APPLICATION, new Note(VALID_NOTE_JANE_STREET));
+        String userInput = targetIndex.getOneBased() + " " + PREFIX_NOTE_INSERT + VALID_NOTE_JANE_STREET;
+        NoteCommand expectedCommand = new NoteInsertCommand(INDEX_FIRST_APPLICATION, new Note(VALID_NOTE_JANE_STREET));
         assertParseSuccess(parser, userInput, expectedCommand);
     }
```

Comment:
Perhaps we should add test cases here for `NoteDeleteCommand` as well?

Reply:
Done